src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  film: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ film }) => (
+  <div className="card">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={film.imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8">{film.title}</p>
+        </div>
+      </div>
+
+      <div className="content">
+        {film.description ? film.description : ''}
```

Comment:
I'd use one of these conditions
```suggestion
        {film.description && ( film.description )} or {film.description || ''}
```

Reply:
Use that in deconstruction with default params to avoid code like that